src/Spryker/Zed/EventBehavior/Persistence/Propel/Schema/spy_event_behavior.schema.xml

```diff
@@ -3,7 +3,7 @@
 
     <table name="spy_event_behavior_entity_change">
         <column name="id_event_behavior_entity_change" type="BIGINT" autoIncrement="true" primaryKey="true"/>
-        <column name="data" type="VARCHAR"/>
```

Comment:
What if we will have the same problem in future? Maybe we should map propel's varchar to mysql's longvarchar then?